src/data/wallets/wallet-filters.ts

```diff
@@ -136,15 +136,15 @@ const walletFilterData: { [key: string]: WalletFilter } = {
     filterKey: "eip_1559_support",
   },
   buy_crypto: {
-    title: "Buy crypto",
+    title: "On ramp to crypto",
     description:
-      "Buy crypto directly in the wallet \n *Note: buying crypto may be region specific",
+      "Buy crypto with fiat directly in the wallet \n *Note: buying crypto may be region specific",
     filterKey: "buy_crypto",
   },
   withdraw_crypto: {
-    title: "Withdraw crypto",
+    title: "Off ramp to fiat",
     description:
-      "Withdraw to fiat directly in the wallet \n *Note: withdrawing crypto may be region specific",
+      "Sell crypto to fiat directly in the wallet \n *Note: withdrawing crypto may be region specific",
```

Comment:
@konopkja from your user testing on this page, do you think it's fair to assume people will understand the what fiat currency is? or should we explain it a little here?

Reply:
for many people who have invested in crypto fiat might be a known term. but i mean if there can be even simpler language than that id welcome it. Any ideas?

Reply:
I'm not so convinced they would. I'll do some testing on friends at the weekend.

Maybe "Sell crypto for fiat (government backed currency) directly in the wallet" or linking to a glossary item in the text?